README.md

```diff
@@ -93,20 +93,21 @@ optional arguments:
   -h, --help            show this help message and exit
   -d DIRECTORY, --debug DIRECTORY
                         path to the debug directory (enables debug mode)
-  -s N, --start N       starting image (default: 0)
-  -f N, --finish N      final image (defaults to until last image)
+  -ff N, --frame_first N       starting image (default: 0)
+  -lf N, --frame_last N      final image (defaults to until last image)
```

Comment:
Arg. Think carefully about your users: if the long version is --frame_last, then the short form should be -fl (**f**rame_**l**ast); -lf is backwards.